_includes/accessibility.html

```diff
@@ -1,18 +1,28 @@
 <section class="usa-prose site-prose margin-top-5">
-  <h2 id="general-accessibility-guidance-for-forms" class="usa-heading heading-margin-alt">General accessibility guidance for forms</h2>
+  <h2 id="general-accessibility-guidance-for-forms" class="usa-heading heading-margin-alt">Accessibility guidance</h2>
```

Comment:
Got the same visual results when removing classes.

```suggestion
  <h2 id="general-accessibility-guidance-for-forms">Accessibility guidance</h2>
```